src/logger.h

```diff
@@ -1,4 +1,28 @@
 #pragma once
+
+#include <fstream>
+
 #include <boost/log/trivial.hpp>
+#include <boost/log/utility/setup/from_stream.hpp>
+#include <boost/log/core/core.hpp>
+#include <boost/log/sources/logger.hpp>
+
+namespace logging = boost::log;
+namespace src = boost::log::sources;
+using LoggerType = src::logger_mt;
+
+#define CREATE_LOCAL_LOGGER(logger) \
```

Comment:
@AGaliuzov, there is also an "add attribute" feature in boost logger. I think we can use it to set logger names via macroses and then output to the log. I really think then it's good to see in the log file something like `[Core.TestRunner][TRACE]: some line`